CHANGELOG.md

```diff
@@ -1,13 +1,36 @@
 # App Center SDK for React Native Change Log
 
-## Version 1.10.0 (Under development)
+## Version 1.10.0
 
+### Features
+
+AppCenterAnalytics
+
+[Android] Add API to specify event persistence priority.
+[IOS] Add new trackEvent APIs that take priority (normal or critical) of event logs. Events tracked with critical flag will take precedence over all other logs except crash logs (when AppCenterCrashes is enabled), and only be dropped if storage is full and must make room for newer critical events or crashes logs.
 ### Fixes
 
 - Fix lint issue on modern projects using latest react-native versions [#451](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/451).
 - Use latest Firebase version [#365](https://github.com/Microsoft/AppCenter-SDK-React-Native/issues/365).
 - [Android] Fix a bug which prevents ErrorAttachmentLog from being send if fileName is not specified.
 
+AppCenter
+- [IOS] Fix an issue where concurrent modification of custom properties was not thread safe.
```

Comment:
Add blankline between section header and list of fixes.
Use `**[iOS]**` (bold, lower case i for iOS).

Reply:
Done